### ISSUE TITLE
Issue with unit tests after adding TypeORM

### ISSUE THREAD

**Issue by vygandas** [cortip/isomera#108]
Currently, we practically disabled unit tests in the API app because it was bottlenecking all other tickets. Decided not to delay TypeORM integration anymore and closed the PR. The problem is, that we have a bunch of new unit tests for API, which were moved from another project, but we never fixed tests that already were not working. It is needed to fix tests and make it all work ok. Some tests might be irrelevant, so making some adjustments is okay.

**Acceptance criteria:**

- All tests are passing in CI and locally (in CI, it's running on affected areas, locally it's all)
- Commands `nx run api:test` and `nx run platform:test` passing successfully with all tests
- All the other checks are passing for the PR (tests, prettier, linting, code quality, etc)

**Comment by jroper-scottlogic** [comment]
I'm looking to contribute to an open source project, could I be assigned this?

**Comment by vygandas** [comment]
@jroper-scottlogic hello. Yes, you can. However, it might be a good idea until I finish another PR #117 which is draft at the moment and very large.

**Comment by vygandas** [comment]
@jroper-scottlogic , allright, massive change is about to be done. Let me know if you are going forward with this ticket. 😊

**Comment by jroper-scottlogic** [comment]
yeah, happy to start now!

**Comment by vygandas** [comment]
@jroper-scottlogic awesome! 👍🏻 Check this out first https://github.com/cortip/isomera/blob/main/CONTRIBUTORS.md

**Comment by vygandas** [comment]
@jroper-scottlogic any luck? :)

**Comment by vygandas** [comment]
I am unasigning @jroper-scottlogic since it was 4 days without any activity.

**Comment by naftalimurgor** [comment]
Hello,
I'd like to work on this as describe and would submit a WIP PR just to set the ball rolling on the issue. Please see the enclosed response on Upwork,
Warm regards,

**Comment by vygandas** [comment]
@naftalimurgor , go for it! 😉

**Comment by naftalimurgor** [comment]
@vygandas  
Let's get this done !

**Comment by demakoff** [comment]
Hey. 
IFAIU the current one already taken by @naftalimurgor and the current Upwork task is not relevant.
@vygandas you can let me know if you will need any further assistance with this project.

**Comment by vygandas** [comment]
@demakoff ah, sorry, forgot to make job private there. I will need more stuff done. First need to get this ready to unblock everything else. I'll take a note to ping you too, when new task appears
